examples/checkSSMobj.py

```diff
@@ -0,0 +1,68 @@
+"""
+This is script will generate basic catalogs from the SSM tables on
+fatboy in order to verify that the interface between SolarSystemObj
+and all of its daughter classes and fatboy is up to date.
+
+This is not included as a unit test, because it depends on a connection
+to fatboy and:
+
+1) the connection to fatboy is still unreliable
+
+2) we have not yet arranged for the Continuous Integration machines (e.g.
+buildbot, Jenkins) to have a connection to fatboy.
+"""
+
+import numpy as np
+import os
+from lsst.utils import getPackageDir
+from lsst.sims.utils import ObservationMetaData
+from lsst.sims.catalogs.measures.instance import InstanceCatalog
+from lsst.sims.catUtils.baseCatalogModels import SolarSystemObj, CometObj, \
+                                                 MBAObj, NEOObj, \
```

Comment:
Good point, let's leave it as that then.

On Fri, Oct 16, 2015, 9:58 AM danielsf <notifications@github.com> wrote:

> In examples/checkSSMobj.py
> <https://github.com/lsst/sims_catUtils/pull/26#discussion_r42264528>:
>
> > +This is not included as a unit test, because it depends on a connection
> > +to fatboy and:
> > +
> > +1) the connection to fatboy is still unreliable
> > +
> > +2) we have not yet arranged for the Continuous Integration machines (e.g.
> > +buildbot, Jenkins) to have a connection to fatboy.
> > +"""
> > +
> > +import numpy as np
> > +import os
> > +from lsst.utils import getPackageDir
> > +from lsst.sims.utils import ObservationMetaData
> > +from lsst.sims.catalogs.measures.instance import InstanceCatalog
> > +from lsst.sims.catUtils.baseCatalogModels import SolarSystemObj, CometObj, \
> > +                                                 MBAObj, NEOObj, \
>
> Do you prefer that? I think I prefer it the way it is since "M", "B", and
> "A" all represent distinct words
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lsst/sims_catUtils/pull/26/files#r42264528>.
>